tests/on_disk_property_storage/test_multiprocess.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2016-2022 Memgraph Ltd. [https://memgraph.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing as mp
+from typing import Optional
+
+import pytest
+import random
+
+from gqlalchemy import Memgraph, Node, Field, SQLitePropertyDatabase
+
+
+db = Memgraph()
+on_disk_db = SQLitePropertyDatabase("./tests/on_disk_storage.db")
+db.add_on_disk_storage(on_disk_db)
```

Comment:
`add_on_disk_storage` seems to be a bit misleading (at least to me), what about `init_disk_storage`?

Reply:
Sure, that sounds good. I was also thinking of
```python
db = Memgraph()
on_disk_db = SQLitePropertyDatabase("./tests/on_disk_storage.db")
on_disk_db.app_init(db)
```
That is used usually in flask based services. What do you think about that?

Reply:
Why not just `init`?

Reply:
Or, just in the constructor?

Reply:
Yeah, Sqlalchemy provides both possibilities:
```python
Sqlalchemy(app)
```
and
```python
db = Sqlalchemy()
db.init_app(app)
```

I don't know why not just `init`, it's a standard with flask based services.
https://stackoverflow.com/questions/58042067/what-is-purpose-of-init-app-function-in-flask

Reply:
I've added the `__init__` approach and renamed the function to `init_disk_storage`